python/ray/train/session.py

```diff
@@ -104,7 +104,10 @@ def finish(self):
         return func_output
 
     def get_next(self) -> Optional[TrainingResult]:
-        """Gets next result from the queue."""
+        """Get the next ``TrainingResult`` from the result queue.
+
+        If the result queue is empty, then this function returns None.
```

Comment:
Wasn't sure whether to put this information under `Returns:` or under the docstring summary. I don't think it really matters, though; I may just be bike-shedding.